### DISCUSSION TITLE
Conditions for matrix blocks fields

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#13533] [Ideas] [answered]
I was recently thinking about how i set up matrix blocks for content builder on my website and thought that using conditions in block fields would allow me to limit numbers of existing block types.
But it seems that there is now way to set up conditions for matrix blocks fields.
Would it be something possible to add in version 4.5 or 5?

**Comment by brandonkelly** [comment, marked as answer]
Coming in 5!

**Reply by brandonkelly** [reply]
Craft 5.0.0-alpha.9 was just released, with live conditional field support 🎉
